### PR TITLE
Iss14 login sign

### PR DESCRIPTION
**Pull request by LPMendoza**
## Description
The design of Sign in, Sign out and Password recovery views was applied.

## How to test it
- Download the project.
- Run the server.
- Go to http://localhost:3000/
- The login screen will be showed.

## Screenshots
image
image
image

### CODE REVIEW

File: app/javascript/pages/TaskPage.js [hunk 1]
```diff
@@ -53,11 +55,15 @@ let TaskPage = ({tasks, handleOnCreate, handleOnUpdate, handleOnDelete}) => {
     }
     return(
         <React.Fragment>
+            <ToastContainer
```
Comment (LDVera) [inline app/javascript/pages/TaskPage.js:58]: :o good job using react

File: app/assets/stylesheets/sessions.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+body {
+    background-image: url("back-circle.svg");
+    background-size: cover;
+}
```
Comment (OchoaHub) [inline app/assets/stylesheets/sessions.scss:4]: Bro, USE 2 SPACES INSTEAD OF TAB

**Review by jnegrete15 — APPROVED** [review]

**Review by LDVera — APPROVED** [review]

**Review by OchoaHub — APPROVED** [review]
Good

File: app/assets/stylesheets/_variables.scss [hunk 3]
```diff
@@ -9,6 +9,6 @@ $small-shadow: 4px 2px 10px rgba(0,0,0, 0.05), -4px -4px 10px rgba(255, 255, 255
 $shadow-task-card: 4px 4px 10px rgba(0, 0, 0, 0.05), -4px -4px 5px rgba(255, 255, 255, 0.7);
 $inner-shadow-task-card: 4px 4px 10px rgba(0, 0, 0, 0), -4px -4px 10px rgba(255, 255, 255, 0), inset 4px 4px 10px rgba(0,0,0, 0.05), inset -4px -4px 10px rgba(255, 255, 255, 0.8);
 $inner-shadow-inputs: inset -4px -4px 4px rgba(255, 255, 255, 0.59), inset 0px 4px 4px rgba(0, 0, 0, 0.05);
-$inner-shadow-buttons: -4px -2px 4px rgba(255, 255, 255, 0), 0px 4px 4px rgba(0, 0, 0, 0), inset -4px -2px 4px rgba(255, 255, 255, 0.25), inset 0px 4px 4px rgba(0, 0, 0, 0.1);
+$inner-shadow-buttons: -4px -2px 4px rgba(255, 255, 255, 0), 0px 4px 4px rgba(0, 0, 0, 0), inset -4px -2px 4px rgba(255, 255, 255, 0.6), inset 0px 4px 4px rgba(0, 0, 0, 0.1);
```
Comment (MiguelOcegueraM) [inline app/assets/stylesheets/_variables.scss:12]: nice inner man

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -159,6 +158,10 @@ h1,h2,h3,h4 {
     box-shadow: $small-shadow !important;
 }
 
+.bg-background {
+    background-color: $bg-white;
+}
+
```
Comment (MiguelOcegueraM) [inline app/assets/stylesheets/application.scss:164]: try to use classes instead of id's

File: app/assets/stylesheets/sessions.scss (hunk 2 above)
Comment (MiguelOcegueraM) [reply app/assets/stylesheets/sessions.scss:4]: agree

**Review by MiguelOcegueraM — COMMENTED** [review]
the code looks great mate, go ahead with the next step!

**Pull request merged**